Project.toml

```diff
@@ -9,7 +9,7 @@ MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 
 [compat]
 JuMP = "~0.20.0"
-MathOptInterface = "~0.9.0"
+MathOptInterface = "=0.9.7"
```

Comment:
Why the equality? Doesn't it work with v0.9.8?

Reply:
> the only fix I could get was to force MOI back to 0.9.7, there seems to be something weird with the mix julia 1.1 and MOI 0.9.8

It seems related to this error

Reply:
I have opened an issue https://github.com/JuliaOpt/MathOptInterface.jl/issues/980

Reply:
It should work with MOI v0.9.9